[Bulk] Validation Fixes (#258)
MicrosoftDocs/bingmaps-docs: BingMaps/sdk-native/getting-started-ios/index.md

---
title: "Getting Started with iOS | Microsoft Docs"

> description: "This article is a tutorial that goes through creating an iOS app with a Bing Maps Native Control step-by-step."

Reviewer: ```suggestion
description: The getting started with iOS article is a tutorial that steps through creating an iOS app with a Bing Maps Native Control.
```
